molecule/backup_restore_migrate/molecule.yml

```diff
@@ -0,0 +1,36 @@
+---
+prerun: false
+dependency:
+  name: galaxy
+  options:
+    requirements-file: molecule/requirements.yml
+driver:
+  name: default
+platforms:
+  - name: instance
+    private_key_file_path: ~/.ssh/id_rsa
+    image_id: ami-0dac29b55bd7c33aa
+    instance_type: m5.large
+    aws_key_name: CI-TEST
+provisioner:
+  name: ansible
+  env:
+    ANSIBLE_ROLES_PATH: "../../roles"
+  config_options:
+    defaults:
+      host_key_checking: false
+      internal_poll_interval: '0.0001'
+    ssh_connection:
+      ssh_args: '-o ControlMaster=auto -o ControlPersist=240s'
+      # note: if we ever need to do some more complex sudo/become usage, pipelining
+      # might need to be disabled, because it messes with these operations
+      pipelining: true
+scenario:
+  test_sequence:
+    - create
+    - prepare
+    - converge
+    - verify
+    - side_effect backup_test.yml destroy_tas.yml restore_test.yml ## Create Backup, Gutting Procedure and then subsequent restore
+    - verify verify_restore.yml
```

Comment:
I'd also suggest adding the idempotence check, to ensure things are stable after the migration.

Reply:
:+1: